### PR TITLE
add github template for PRs

### PR DESCRIPTION
**Pull request by ndushay**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Why was this change made?
+
+
+
+## Was the API documentation (openapi.json) updated?
```
Comment (jmartin-sul) [inline .github/pull_request_template.md]: could we change this to `## Was the usage documentation (e.g. wiki, README, queue or DB specific README) updated?`  there's not yet an openapi.json file for this project, but i think adding one would be a good idea, and i'd be happy to ticket that.  should be easy while things are still small.

i do really like adding this template though.

Comment (ndushay) [reply .github/pull_request_template.md]: sure - i'll make your change;  you make the ticket.

Comment (ndushay) [reply .github/pull_request_template.md]: done.

Comment (jmartin-sul) [reply .github/pull_request_template.md]: thanks, and done:  https://github.com/sul-dlss/preservation_catalog/issues/1216

**Pull request merged**